docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -0,0 +1,176 @@
+PKI Quick Installation Guide
+============================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+
+This guide has been created to give a quick start to install the PacketFence PKI in PacketFence 5.2+. This guide does not include advanced troubleshooting of EAP-TLS connection. Refer to the documentation of EAP-TLS, RADIUS and OpenSSL for advanced features.
+
+Assumptions
+-----------
+
+* You have at least one server with PacketFence 5.2+
+* The server already have a properly configured switch with 802.1X support
+* PacketFence RADIUS server is working in your environnement
+* PacketFence management IP will be 192.168.1.5 and has useStrongerSecret as it's RADIUS shared secret
+
+Installation
+------------
+
+Step 1: Install the PKI
+~~~~~~~~~~~~~~~~~~~~~~~
+
+NOTE: PacketFence PKI is available on the PacketFence repository as the package "packetfence-pki". This is available for CentOS6/7 and Debian7/8.
+
+Installing 
+^^^^^^^^^^
+
+First of all, you will need to install the package, use the following command matching your OS.
+
+Debian/Ubuntu:
+ # sudo apt-get install packetfence-pki
+
+CentOS/RHEL:
+ # yum install packetfence-pki --enablerepo=PacketFence
+
+Step2: Configuring the PKI
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PacketFence-PKI Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+NOTE: This following configuration will allow you to generate certificates for an EAP-TLS configuration, if you want to use the PKI for another purpose you will have to find the Key Usage/Extended Key Usage necessary.
+
+Now that your PKI is installed, you need to configure it.
+
+PKI Side
+^^^^^^^^
+
+You will first need to log into the PKI:
+
+* Address: https://192.168.1.5:9191
+* Login: admin
+* Password: packetfence
+
+After you logged in there are few steps to follow to have the PKI working for an EAP-TLS configuration with PacketFence. You will need to create the following items: 
+
+* Certificate of Authority.
+* Profiles(for each purpose, ie Server/Client auth are 2 different profiles).
+* Manual certificate generation.(for the RADIUS server)
+
+To help you out through the configuration, the first login will ask you to create a Certificate of Authority.
+
+Certificate of Authority
+^^^^^^^^^^^^^^^^^^^^^^^^
+ 
+* Key type, size and digest: we advise to use the following RSA, 2048, sha1.
+* Key Usage and Extended Key Usage can be filled if necessary, more information about Key Usage and Extended Key Usage are available from the RFC5280, parts 4.2.1.3 and 4.2.1.12.
```

Comment:
Going all @louismunro by refering people to the RFC.

You could at least supply the key usage when doing EAP-TLS as this guide is made for that.